classes/system/linux/system/repo/openstack.yml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  linux:
+    system:
+      repo:
+        mirantis_openstack:
+          source: "deb http://mirror.fuel-infra.org/mos-repos/ubuntu/8.0-mu-2/ mos8.0 main restricted"
+          architectures: amd64
+          key_url: "http://mirror.fuel-infra.org/mos-repos/ubuntu/8.0-mu-2/archive-mos8.0.key"
+        mirantis_openstack_hotfix:
+          source: "deb http://mirror.fuel-infra.org/mos-repos/ubuntu/8.0-mu-2/ mos8.0-hotfix main restricted"
+          architectures: amd64
+          key_url: "http://mirror.fuel-infra.org/mos-repos/ubuntu/8.0-mu-2/archive-mos8.0.key"
+        mirantis_openstack_proposed:
+          source: "deb http://mirror.fuel-infra.org/mos-repos/ubuntu/8.0-mu-2/ mos8.0-proposed main restricted"
+          architectures: amd64
+          key_url: "http://mirror.fuel-infra.org/mos-repos/ubuntu/8.0-mu-2/archive-mos8.0.key"
+        mirantis_openstack_security:
+          source: "deb http://mirror.fuel-infra.org/mos-repos/ubuntu/8.0-mu-2/ mos8.0-security main restricted"
+          architectures: amd64
+          key_url: "http://mirror.fuel-infra.org/mos-repos/ubuntu/8.0-mu-2/archive-mos8.0.key"
+        mirantis_openstack_updates:
+          source: "deb http://mirror.fuel-infra.org/mos-repos/ubuntu/8.0-mu-2/ mos8.0-updates main restricted"
+          architectures: amd64
+          key_url: "http://mirror.fuel-infra.org/mos-repos/ubuntu/8.0-mu-2/archive-mos8.0.key"
```

Comment:
What about the mos8.0-holdback distribution? I think this is used currently in MOS/Fuel deployments.